### PR TITLE
V2 Updated test helper for rendering content with textstring

### PR DESCRIPTION
**Pull request by nhudinh0309**

### CODE REVIEW

File: lib/helpers/DocumentTypeApiHelper.ts [hunk 1]
```diff
@@ -597,4 +597,34 @@ export class DocumentTypeApiHelper {
...
+        .withDataTypeId(dataTypeId)
+        .done()
+      .addAllowedTemplateId()
+        .withId(templateId)
+        .done()
+      .withDefaultTemplateId(templateId)
+      .build();
+    
+    const documentTypeId =  await this.create(documentType);
+    return documentTypeId === undefined ? '' : documentTypeId;
```
Comment (andr317c) [inline lib/helpers/DocumentTypeApiHelper.ts:628]: Is this necessary?

File: lib/helpers/ContentRenderUiHelper.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+export class ContentRenderUiHelper extends UiBaseLocators {
+  private readonly contentRenderValue: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.contentRenderValue = page.locator('[data-mark="content-render-value"]');
+  }
+
+  async goToContentRenderPage(contentURL: string) {
```
Comment (andr317c) [inline lib/helpers/ContentRenderUiHelper.ts:13]:
```suggestion
  async navigateToRenderedContent(contentURL: string) {
```

**Review by andr317c — CHANGES_REQUESTED** [review]
Had a few comments

File: lib/helpers/DocumentTypeApiHelper.ts (hunk 1 above)
Comment (nhudinh0309) [reply lib/helpers/DocumentTypeApiHelper.ts:628]: Yes, the output of this function is the input for the createPublishedDocumentWithValue function, and in that function, documentTypeId should not be undefined. To ensure consistency in the code, I will move this logic from the current function to the createPublishedDocumentWithValue function.

**Review by andr317c — APPROVED** [review]
LGTM

**Pull request merged**